keps/sig-cloud-provider/20190125-out-of-tree-openstack.md

```diff
@@ -83,7 +83,7 @@ OpenStack Cloud Provider is well tested, CI running at [OpenLab](https://github.
 
 ### Graduation Criteria
 
-This feature is complete, well tested, in parity with intree openstack provider. Documents needs to be updated as per sig-cloud-provider guideline. 
+This feature is complete, well tested, in parity with intree openstack provider. Documention regarding migration from in-tree cloud provider to openstack-cloud-controller-manager can be found [here](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/migrate-to-ccm-with-csimigration.md)
```

Comment:
Are there also general docs on installing the out-of-tree openstack provider?

Reply:
@andrewsykim yes, we have documentation regarding installing ccm using kubeadm and also getting started guide on installing the provider on configuration needed for using external provider. https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-controller-manager-with-kubeadm.md , https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/getting-started-provider-dev.md